src/img/icons/circle-thumbs-up.svg

```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 30 30" fill="none">
+<rect opacity="0.6" x="26.5" y="26.5" width="26" height="26" rx="13" transform="rotate(-180 26.5 26.5)" fill="white" stroke="#68D2D2"/>
+<path d="M14.8326 11.0025L14.8326 8.57249C14.8326 8.08913 14.6375 7.62558 14.2902 7.2838C13.9429 6.94201 13.4719 6.75 12.9807 6.75L10.5115 12.2175L10.5115 18.8999L17.4746 18.8999C17.7724 18.9032 18.0613 18.8005 18.2881 18.6108C18.515 18.421 18.6646 18.1569 18.7092 17.8672L19.5611 12.3997C19.5879 12.2256 19.576 12.0478 19.5261 11.8786C19.4762 11.7095 19.3895 11.553 19.2721 11.4201C19.1547 11.2872 19.0093 11.181 18.846 11.1089C18.6827 11.0368 18.5054 11.0005 18.3265 11.0025L14.8326 11.0025ZM10.5115 18.8999L8.65964 18.8999C8.3322 18.8999 7.73375 18.8999 7.42508 18.8999C7.4251 18.5962 7.42505 18.0072 7.42505 17.6849L7.42505 13.4325C7.42505 13.1102 7.42513 12.825 7.42513 12.2175C8.04242 12.2175 8.33221 12.2175 8.65964 12.2175L10.5115 12.2175" fill="#68D2D2"/>
+<path d="M10.5115 12.2175L12.9807 6.75C13.4719 6.75 13.9429 6.94201 14.2902 7.2838C14.6375 7.62558 14.8326 8.08913 14.8326 8.57249L14.8326 11.0025L18.3265 11.0025C18.5054 11.0005 18.6827 11.0368 18.846 11.1089C19.0093 11.181 19.1547 11.2872 19.2721 11.4201C19.3895 11.553 19.4762 11.7095 19.5261 11.8786C19.576 12.0478 19.5879 12.2256 19.5611 12.3997L18.7092 17.8672C18.6646 18.1569 18.515 18.421 18.2881 18.6108C18.0613 18.8005 17.7724 18.9032 17.4746 18.8999L10.5115 18.8999M10.5115 12.2175L10.5115 18.8999M10.5115 12.2175L8.65964 12.2175C8.33221 12.2175 8.04242 12.2175 7.42513 12.2175C7.42513 12.825 7.42505 13.1102 7.42505 13.4325L7.42505 17.6849C7.42505 18.0072 7.4251 18.5962 7.42508 18.8999C7.73375 18.8999 8.3322 18.8999 8.65964 18.8999L10.5115 18.8999" stroke="#F9FAFA" stroke-linecap="round" stroke-linejoin="round"/>
+</svg>
```

Comment:
Same as above, the icon is poorly optimized. I've re-made it, so please use this file instead:
```svg
<?xml version="1.0" encoding="UTF-8" standalone="no"?><svg viewBox="0 0 30 30"><path d="M 15,0 C 6.6996097,0 0,6.6996092 0,15 0,23.30039 6.6996097,30 15,30 23.300391,30 30,23.30039 30,15 30,6.6996092 23.300391,0 15,0 Z m 0,1.1113281 C 22.704055,1.1113281 28.888672,7.2959456 28.888672,15 28.888672,22.704054 22.704055,28.888672 15,28.888672 7.2959461,28.888672 1.1113281,22.704054 1.1113281,15 1.1113281,7.2959456 7.2959461,1.1113281 15,1.1113281 Z" style="color:#000000;font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:medium;line-height:normal;font-family:sans-serif;font-variant-ligatures:normal;font-variant-position:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-alternates:normal;font-feature-settings:normal;text-indent:0;text-align:start;text-decoration:none;text-decoration-line:none;text-decoration-style:solid;text-decoration-color:#000000;letter-spacing:normal;word-spacing:normal;text-transform:none;writing-mode:lr-tb;direction:ltr;text-orientation:mixed;dominant-baseline:auto;baseline-shift:baseline;text-anchor:start;white-space:normal;shape-padding:0;clip-rule:nonzero;display:inline;overflow:visible;visibility:visible;opacity:0.6;isolation:auto;mix-blend-mode:normal;color-interpolation:sRGB;color-interpolation-filters:linearRGB;solid-color:#000000;solid-opacity:1;vector-effect:none;fill:#fe5e7c;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:1.11111116;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;color-rendering:auto;image-rendering:auto;shape-rendering:auto;text-rendering:auto;enable-background:accumulate" /><path d="m 13.519333,17.775 v 2.7 c 0,0.537111 0.216778,1.052111 0.602667,1.431889 C 14.507889,22.286667 15.031222,22.5 15.577,22.5 l 2.743556,-6.075 V 9.0000884 H 10.583767 C 10.252944,8.9964106 9.9319111,9.1105106 9.6798333,9.3213773 9.4277556,9.5322439 9.2616,9.8256662 9.212,10.147577 l -0.9465222,6.074979 c -0.029844,0.193444 -0.016578,0.391 0.038867,0.579 0.055433,0.187888 0.1517334,0.361777 0.2822223,0.509444 0.1304889,0.147667 0.2920333,0.265667 0.4734555,0.345778 0.1814222,0.08011 0.3783778,0.120444 0.5772222,0.118222 z m 4.801223,-8.7749116 h 2.057666 c 0.363778,0 1.028778,0 1.371667,0 0,0.3375 1.11e-4,0.9919444 1.11e-4,1.3499886 V 15.075 c 0,0.358111 -1.11e-4,0.675 -1.11e-4,1.35 -0.685889,0 -1.007889,0 -1.371667,0 h -2.057666" style="fill:#fe5e7c;stroke-width:1.11111116" /><path d="m 18.3125,8.4375 a 0.55561114,0.55561114 0 0 0 -0.08984,0.00781 H 10.589844 C 10.12731,8.4401705 9.6786979,8.5980043 9.3242188,8.8945312 8.9697515,9.1910482 8.7324084,9.6062496 8.6621094,10.0625 l -0.9453125,6.074219 a 0.55561114,0.55561114 0 0 0 0,0.002 c -0.042318,0.274296 -0.023826,0.554092 0.054687,0.820312 0.078523,0.26615 0.2144544,0.5125 0.3984375,0.720704 0.1838999,0.208109 0.4116633,0.37206 0.6660156,0.484374 0.2543064,0.112295 0.5285868,0.169123 0.8066406,0.166016 h 3.3203129 v 2.144531 c 0,0.686593 0.278819,1.345185 0.769531,1.828125 0.490732,0.482961 1.153202,0.751953 1.84375,0.751954 a 0.55561114,0.55561114 0 0 0 0.507812,-0.326172 l 2.595704,-5.748047 H 20.378906 21.75 a 0.55561114,0.55561114 0 0 0 0.554688,-0.554688 c 0,-0.674939 0,-0.993336 0,-1.351562 v -4.72461 c 0,-0.3581586 -10e-7,-1.0122308 0,-1.349609 A 0.55561114,0.55561114 0 0 0 21.75,8.4453125 H 20.378906 18.421875 A 0.55561114,0.55561114 0 0 0 18.3125,8.4375 Z m -7.734375,1.1171875 a 0.55561114,0.55561114 0 0 0 0.0059,0 h 7.181641 v 6.7500005 l -2.492187,5.517578 c -0.274885,-0.06171 -0.560579,-0.112592 -0.761719,-0.310547 -0.281067,-0.276616 -0.4375,-0.64948 -0.4375,-1.03711 V 17.775391 A 0.55561114,0.55561114 0 0 0 13.519531,17.21875 H 9.6367188 a 0.55561114,0.55561114 0 0 0 -0.00586,0 c -0.119635,0.0013 -0.2371651,-0.02238 -0.3457032,-0.07031 C 9.1766641,17.10053 9.079031,17.030584 9.0019531,16.943359 8.9249584,16.856229 8.8702342,16.754158 8.8378906,16.644531 8.8055148,16.534752 8.7970825,16.419233 8.8144531,16.306641 l 0.9472657,-6.074219 c 0.028901,-0.187571 0.1257024,-0.3591589 0.2753902,-0.4843751 0.149677,-0.1252065 0.341904,-0.195573 0.541016,-0.1933594 z m 8.298828,0 h 1.501953 0.814453 c 3.2e-5,0.2912267 0.002,0.5841005 0.002,0.7949215 v 4.72461 c 0,0.210556 -0.0019,0.531555 -0.002,0.794922 h -0.814453 -1.501953 z" style="color:#000000;font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:medium;line-height:normal;font-family:sans-serif;font-variant-ligatures:normal;font-variant-position:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-alternates:normal;font-feature-settings:normal;text-indent:0;text-align:start;text-decoration:none;text-decoration-line:none;text-decoration-style:solid;text-decoration-color:#000000;letter-spacing:normal;word-spacing:normal;text-transform:none;writing-mode:lr-tb;direction:ltr;text-orientation:mixed;dominant-baseline:auto;baseline-shift:baseline;text-anchor:start;white-space:normal;shape-padding:0;clip-rule:nonzero;display:inline;overflow:visible;visibility:visible;opacity:1;isolation:auto;mix-blend-mode:normal;color-interpolation:sRGB;color-interpolation-filters:linearRGB;solid-color:#000000;solid-opacity:1;vector-effect:none;fill:#f9fafa;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:1.11111116;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;color-rendering:auto;image-rendering:auto;shape-rendering:auto;text-rendering:auto;enable-background:accumulate" /></svg>
```

As above, note that the above changes the size to actually be `30x30` so you might need to adjust your styles